packages/taquito/src/read-provider/interface.ts

```diff
@@ -112,6 +112,16 @@ export interface TzReadProvider {
     block: BlockIdentifier
   ): Promise<SaplingDiffResponse>;
 
+  /**
+   * @description Access the sapling state of a smart contract.
+   * @param contractAddress Address of the smart contract
```

Comment:
nit "contractAddress ~Address~ of the smart contract"

Reply:
not sure if we need two addresses there

Reply:
@zainen since `contractAddress` is the variable name I think it makes sense

Reply:
yes "contractAddress" is the name of the param and "Address of the smart contract" is the description

Reply:
It looks like this with typedoc:
![image](https://user-images.githubusercontent.com/65877347/184023695-609e2af9-3872-4ffd-8654-a4f76e77792e.png)

Reply:
maybe contractAddress the address of the smart contract?

Reply:
the address address just made me do a double take :O

Reply:
ok i will add "the"